docs proofreading - deploy and setup pages

arouyan opened the pull request:
@johnnyaug I went through the content in deploy and setup categories in the docs, made some slight corrections.

johnnyaug commented on docs/setup/hooks.md:
```diff
@@ -121,20 +121,20 @@ When an actionable event (see Supported Events above) takes place, lakeFS will r
 {: .no_toc }
 
-There are 2 types of files that are stored in the metadata section of lakeFS repository with each `Run`:
+The metadata section of lakeFS repository with each `Run` stored two types of files:
```
```suggestion
The metadata section of lakeFS repository with each `Run` contains two types of files:
```

johnnyaug commented on docs/setup/import.md:
```diff
@@ -87,9 +87,9 @@ lakectl ingest \
-environment variables and credential files [in the same way that the AWS cli does](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html){: target="_blank" }
-Specify an endpoint to ingest from other S3 compatible storage solutions, e.g. add `--s3-endpoint-url https://play.min.io`.
+The `lakectl ingest` command will attempt to use the current user's existing credentials and respect instance profiles,
+environment variables, and credential files [similarly to AWS cli does](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html){: target="_blank" }
```
```suggestion
environment variables, and credential files ([similarly to AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html){: target="_blank" })
```

johnnyaug commented on docs/setup/import.md:
```diff
@@ -200,7 +200,7 @@ lakefs import --with-merge lakefs://example-repo -m s3://example-bucket/path/to/
 1. You can download the `lakefs` binary from [here](https://github.com/treeverse/lakeFS/releases). Make sure you choose one compatible with your installation of lakeFS.
 
-1. Use a configuration file like the one used to start your lakeFS installation. This will be used to access your database. An example can be found [here](../reference/configuration.html#example-aws-deployment).
+1. Use a configuration file like the one used to start your lakeFS Installation. This will be used to access your database. An example can be found [here](../reference/configuration.html#example-aws-deployment).
```
Why?

arouyan replied:
I've seen installation written with a capital letter somewhere in the docs and assume this is how it should be, some kind of lakeFS standard. If not, then there's definitely no reason to capitalize it

johnnyaug replied:
I think it was a typo

johnnyaug commented on docs/setup/index.md:
```diff
@@ -1,10 +1,10 @@
 ---
 layout: default
-title: Setup lakeFS
+title: Set up lakeFS
```
This is a parent of other pages: it needs to be changed in the parent directive of all its children

johnnyaug replied:
I'm not a native English speaker so I don't know which one is better. My point is that this way the subpages will not appear - so the PR either contains the whole thing or none.

johnnyaug commented on docs/setup/storage/blob.md:
```diff
@@ -10,23 +10,23 @@ has_children: false
 {: .no_toc }
 
-In case you want your lakeFS Installation (we will install in the next step) to access this Container using Active Directory authentication,
+In case you want your lakeFS Installation to access this Container using Active Directory authentication (we will install in the next step),
 First go to the container you created in step 1.
```
```suggestion
In case you want your lakeFS Installation to access this Container using Active Directory authentication,
first go to the container you created in step 1.
```

johnnyaug commented on docs/setup/storage/s3.md:
```diff
@@ -11,10 +11,10 @@ has_children: false
-   1. Block public access
-   1. Disable Object Locking
-1. lakeFS requires permissions to interact with your bucket. Following is a minimal bucket policy. To add it, go to the `Permissions` tab, and paste it as :
+1. Make that sure you:
```
```suggestion
1. Make sure that you:
```

johnnyaug reviewed (changes requested):
Thanks for this change! See my comments.
Also, I see no reason to write "installation" with a capital i.

johnnyaug commented on docs/setup/import.md:
```diff
@@ -48,12 +48,12 @@ Once the import is completed, you can merge the changes from the import branch t
 
 Import is feasible only from source object storage that matches the storage namespace of the
-current repository.  lakeFS S3 installation cannot import from Azure bucket.
+current repository. The lakeFS S3 Installation cannot import from Azure bucket.
```
This sentence should be:

_Importing is only possible from the object storage service in which your installation stores its data. For example, if lakeFS is configured on top of S3, you cannot import data from Azure._

johnnyaug commented on docs/understand/roadmap.md:
```diff
@@ -18,101 +18,99 @@ redirect_from: ../roadmap.html
-In those cases, a merge operation might have to be aware of the structure of the data: 
+However, in some cases simply creating a union of object modifications from both references isn't good enough. 
+Modern table formats such as Delta Lake, Hudi, and Iceberg rely on a set of manifest or log files that describe the logical structure of the table. 
+In those cases, a merge operation might have to be aware of the data structure: 
```
Data structure is an overloaded term - "the structure of the data" makes more sense here.

arouyan replied:
Fixed

johnnyaug commented on docs/use_cases/iso_env.md:
```diff
@@ -110,7 +110,7 @@ As expected, there are now twice as many rows, 972k, on the `double-branch` bran
 Data duplication introduce errors into our data analytics, BI and machine learning efforts, hence we would like to avoid duplicating our data.
 
-Do be afraid! 
+There's lots to afraid of! 
```
Is there? 👹 
i think let's remove this sentence.

johnnyaug reviewed (changes requested):
1. Still capital I's in the word "installation"
2. Changed the title of a page to "Set up lakeFS", but not the children pages: this will not work.
3. Unfortunately, docs/reference/configuration.md is generated automatically from our code. You cannot change it here. Sorry about that.

arouyan commented:
Thanks a lot for your feedback! I removed all the capitalized instances of "installations" and changed the children pages in the Set up lakeFS section to match the parent.
